### DISCUSSION TITLE
Feature request: onCellDblClicked event

### DISCUSSION THREAD

**Discussion by akoltun** [glideapps/glide-data-grid#428] [Ideas]
There is `onCellClicked` event.
It would be nice to have also `onCellDblClicked` for mouse double click.

For example, in my use case, when user double clicks on the cell of a report, the details view should be opened for this cell.

**Comment by jassmith** [comment]
Im guessing you have the overlay disabled for these cells?

**Reply by akoltun** [reply]
Yes, overlay is disabled.
Honestly, I've never used it yet. I suppose it is used for editing.

As a workaround, I implemented double click in this way:
```
  const cellClickTimeRef = useRef<number>(0)
  const cellClicked = useCallback(
    (item: Item, event: CellClickedEventArgs) => {
        if (onCellClicked) onCellClicked(item, event)

        const time = Date.now()
        if (onCellDblClicked && time - cellClickTimeRef.current < 300) onCellDblClicked(item, event)
        cellClickTimeRef.current = time
    },
    [onCellClicked, onCellDblClicked],
  )
```
